std/concurrency.d

```diff
@@ -1013,21 +947,20 @@ private void unregisterMe()
  *  true if the name is available and tid is not known to represent a
  *  defunct thread.
  */
-bool register( string name, Tid tid )
+bool register(string name, Tid tid)
 {
-    synchronized( registryLock )
+    synchronized (registryLock)
```

Comment:
Q: Should we enforce a whitespace after `synchronized`  within the entire Phobos codebase by adding this to the checks to the `posix.mak`?